gargoyle/models.py

```diff
@@ -214,7 +219,7 @@ def get_active_conditions(self, manager):
         """
         Returns a generator which yields groups of lists of conditions.
 
-        >>> for label, set_id, field, value, exclude in gargoyle.get_all_conditions():
+        >>> for label, set_id, field, value, exclude, condition_type in gargoyle.get_all_conditions():
         >>>     print("%(label)s: %(field)s = %(value)s (exclude: %(exclude)s)" % (label, field.label, value, exclude))
```

Comment:
`condition_type` missing from the comment

Reply:
I think the comment is just an example about how to use the generator... in order to make the example work we just need to add `condition_type` in the `for loop` statement.